packages/core/src/features/cohort/availableCohortsSlice.ts

```diff
@@ -1033,3 +1031,65 @@ export const divideCurrentCohortFilterSetFilterByPrefix = (
 
   return divideFilterSetByPrefix(cohort?.filters, prefixes);
 };
+
```

Comment:
Was this to fix the circular import warning?

Reply:
Yes. should've kept the const in different file.

Reply:
That warning was driving me crazy 😄